README.md

```diff
@@ -16,7 +26,7 @@ The frontend of the same project can be found [here](https://github.com/Girl-Cod
 6. [Contributing](#contributing)
 7. [Copyright and license](#copyright-and-license)
 
-<h2 align="center">Install</h2>
+## Install <img src="https://github.com/Samridhi-98/Images/blob/master/Images/test.svg" width="3%">
```

Comment:
These images are not needed, and if you want to add them, they should be added in the assets folder of this project and in not your account.